a10_neutron_lbaas/tests/unit/v1/test_handler_hm.py

```diff
@@ -44,31 +44,38 @@ def fake_hm(self, type):
 
     def test_create_ping(self):
         self.a.hm.create(None, self.fake_hm('PING'), 'p01')
+        time.sleep(5)
```

Comment:
This was done not to make it appear to run, but because it takes a good amount of time for the request to be made, added to the queue, and the given operation to preform. However, I am currently working on decoupling the handlers from fake_a10openstack_lb to test them separately from the thread.